doc/rst/developer/bestPractices/TestAnnotationsLocally.rst

```diff
@@ -1,3 +1,5 @@
+.. _best-practices-test-annotations-locally:
```

Comment:
Not necessarily your problem, but I wanted to point out that this still document still mentions Travis.  If you don't want to handle that in this PR, it'd be good to open an issue tracking developer doc updates that lists it.

Reply:
I've changed the mention to GitHub CI.